### PR TITLE
Use main instead of master for the main branch

### PR DESCRIPTION
**Pull request by coderbyheart**
I propose to use `main` as the name for the main branch, instead of `master`.

If accepted this I will push a new branch based off `master` as `main` and make it the default branch, afterwards I will delete `master`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # ![Favicon](target/apple-icon-57x57.png) Software Craft Website
-[![Build Status](https://travis-ci.org/softwarecrafters/website.svg?branch=master)](https://travis-ci.org/softwarecrafters/website) [![Deps](https://david-dm.org/softwarecrafters/website.svg)](https://david-dm.org/softwarecrafters/website) [![DevDeps](https://david-dm.org/softwarecrafters/website/dev-status.svg)](https://david-dm.org/softwarecrafters/website)
+[![Build Status](https://travis-ci.org/softwarecrafters/website.svg?branch=saga)](https://travis-ci.org/softwarecrafters/website) [![Deps](https://david-dm.org/softwarecrafters/website.svg)](https://david-dm.org/softwarecrafters/website) [![DevDeps](https://david-dm.org/softwarecrafters/website/dev-status.svg)](https://david-dm.org/softwarecrafters/website)
```
Comment (jukecraft) [inline README.md:2]: Great suggestion! Did you want to name it `main` or `saga`?

Comment (coderbyheart) [reply README.md]: Right! I was using my goto alternative.

**Comment by jukecraft** [comment]
@coderbyheart approved! Can you merge yourself or do you need me to? Also, should we delete the master branch?

**Comment by coderbyheart** [comment]
> @coderbyheart approved! Can you merge yourself or do you need me to? Also, should we delete the master branch?

I can do this myself.

I will push a new branch based off `master` as `main` and make it the default branch, afterwards I will delete `master`.

**Comment by rradczewski** [comment]
I'll need to change the deployment on netlify to use the `main` branch, but that's a quick fix. Just ping me once this is taken care of (or once the `main` branch exists, that's sufficient).

**Comment by coderbyheart** [comment]
@rradczewski @jukecraft I don't have the permissions to change the default branch to `main`.

It exists now, so you need to do this.

**Comment by rradczewski** [comment]
image

Done, but looks like an admin can also rename the main branch in the UI (I assume for these exact scenarios). Good to know in the future! Thanks @coderbyheart in any case :+1:

**Pull request merged**